tailwind.config.js

```diff
@@ -54,6 +54,7 @@ module.exports = {
           light: '#F19E7A',
           solid: '#D96F41',
           dark: '#AC5028',
+          pending: '#FFA216',
```

Comment:
This isn't the same yellow/orange Hexadecimal value as in the Figma, can you please mate it to the Figma?

Reply:
this is not part of this PR is must have came from dashboard branch merge